### ISSUE TITLE
Line breaks are removed from selected text on Google Translate

### ISSUE THREAD

**Issue by GoogleCodeExporter** [avm99963/translateselectedtext#4]
```
What steps will reproduce the problem?
1. Select text containing a line break.
2. Right click, select "Translate selection into..." and select any language.

What is the expected output? What do you see instead?
The line breaks should be printed on the translation box. Instead, they are 
ignored and replaced with spaces.

Additional information:
This bug was filled to escalate a bug report on the CWS listing support tab 
written by Dustin Falgout.

Actually, the extension behaves like this due to Chromium bug 116429: 
https://code.google.com/p/chromium/issues/detail?id=116429

Comments #4, #6 and #11 on that Chromium bug suggest a workaround for this 
issue, but unfortunately the workarounds requires the extension to ask for the 
permission of being able to read the entire webpage when the user wants to 
translate the text selection and this is against this developer core value 
which is that all extensions should request as little permissions as possible. 
This developer thinks that requesting the tabs or activeTab permissions is not 
worth in order to solve this little issue and therefore, I'll leave this issue 
as WontFix, and all will be working as intended when the Chromium bug is fixed.
```

Original issue reported on code.google.com by `jocdelad...@gmail.com` on 30 Nov 2014 at 7:11

**Comment by avm99963** [comment]
Reopening this issue, since it was incorrectly autoclosed when migrating from Google Code to GitHub.

The status of this bug was `ExternalDependency`, because before we can fix this in the extension the following Chromium bug must be fixed: https://crbug.com/116429
